fc/cli/reports.py

```diff
@@ -22,22 +23,25 @@ def usertasks(username: str):
     auth = ComboAuth(username)
 
     # setup and build credentials for google api calls
-    SCOPES = ['https://www.googleapis.com/auth/spreadsheets', 'https://www.googleapis.com/auth/drive.file']
-    SERVICE_ACCOUNT_FILE = auth.google_service_acct_creds()
-    SHEET_CREATE_URL = auth.sheet_create_url()
+    g_scopes = None
+    g_service_acct_file = None
+    g_sheet_create_url = None
```

Comment:
All the `None` assignments here can be removed since you are setting them just one line below.